react/localState.js

```diff
@@ -211,12 +234,24 @@ export default function(client) {
     __typename: 'OrderFormStorePreferencesData',
   })
 
+  const mapItemOptions = option => ({
+    seller: null,
+    assemblyId: null,
+    id: null,
+    quantity: null,
+    ...option,
+    __typename: 'MinicartItemOptions',
+  })
+  
+
   const mapToMinicartItem = item => ({
     seller: null,
     index: null,
     parentItemIndex: null,
     parentAssemblyBinding: null,
     ...item,
+    options: (item.options || []).map(mapItemOptions),
+    assemblyOptions: mapAssemblyOptions(item.assemblyOptions || {}),  
```

Comment:
With this, we will always have some `item.assemblyOptions`. What about this case?
```js
if (item.assemblyOptions) {
  // ...
}
```